run_sorting_hat_test_data.sample.sh

```diff
@@ -103,7 +115,7 @@ tail -f $LOG_PATH &
 start=`date +%s`
 echo 'starting bulk '$GOAL'...'
 
-snakemake --use-conda --conda-frontend mamba \
+snakemake --slurm --use-conda --conda-frontend mamba \
```

Comment:
This `--slurm` should be replaced with a variable which set at line 35 either as --slurm or empty in case of LSF